src/components/tabs/tabs.ts

```diff
@@ -79,6 +88,19 @@ export class MdTabGroup {
     return this._selectedIndex;
   }
 
+  /**
+   * Determines if an index is valid.  If the tabs are not ready yet, we assume that the user is
+   * providing a valid index and return true.
+   */
+  isValidIndex(index: number): boolean {
+    if (this._tabs) {
+      const tab = this._tabs.toArray()[index];
+      return tab && !tab.disabled;
+    } else {
+      return true;
```

Comment:
If you don't have any tabs, why is it `true` instead of `false`? Wouldn't any index be "invalid", then?